backend/tests/test_quiz.py

```
@@ -0,0 +1,373 @@
+    assert response.status_code == 404
+
+
+@pytest.mark.asyncio
+async def test_create_quiz_long_title(async_client: AsyncClient, session: AsyncSession):
+    """Tests that creating a quiz with a title > 50 chars fails with a 422 error."""
+    authenticated_client = await get_authenticated_client(async_client, session)
+    long_title = "a" * 51
+    response = await authenticated_client.post(
+        "/api/quizzes/", json={"title": long_title}
+    )
+    assert response.status_code == 422
```

Ruff (0.12.2)

Use of `assert` detected

(S101)